command/task_create.go

```diff
@@ -79,15 +82,12 @@ func (c *taskCreateCommand) Synopsis() string {
 
 // Run runs the command
 func (c *taskCreateCommand) Run(args []string) int {
-	c.meta.setFlagsUsage(c.flags, args, c.Help())
```

Comment:
Just wanted to double-check removing this `setFlagsUsage()`. It looks like it causes the CLI to print the help text when someone tries to pass in an invalid flag

e.g.

```
build/consul-terraform-sync task create -task=task_file.hcl
==> Error: unsupported arguments in flags '-task=task_file.hcl'
    Please see --help information below for supported options:

Usage: consul-terraform-sync task create [options] -task-file=<task config>
...
```

Reply:
ah, yes, I'll add it back